src/server.py

```diff
@@ -38,6 +38,7 @@ def __init__(self, uuid_, metadata, sender, metadata_extras=None):
         self._stop_loop = False
         self._n_loop = 0
         self._is_initial_seek = False
+        self._paused_by_service = False
```

Comment:
Is this variable really necessary? You could use the `GstElement` internal state and create a `paused` property (I'm not sure what `_by_service` mean)

Reply:
Can't really do that, since there might be other ways in which a sound could be in the paused state. That attribute/property helps to differentiate between those and the ones that were actually paused by the "priority" mechanism (specially important at the moment of resuming).